src/api/hooks/useFunds.ts

```diff
@@ -18,39 +19,57 @@ export const CROWD_PREFIX = stringToU8a('modlpy/cfund');
 
 export type ChannelMap = Record<string, [HrmpChannelId, HrmpChannel][]>;
 
-export default function useFunds() {
+export function useFunds() {
   const bestNumber = useBestNumber();
 
   return useApiQuery(
-    ['funds'],
+    ['parachain_crowdloan_funds'],
     async (api) => {
       const paraIdKeys = await api.query.crowdloan?.funds?.keys<[ParaId]>();
       const paraIds = paraIdKeys ? extractFundIds(paraIdKeys) : [];
 
-      const [rawFunds, rawLeases]: [
-        Option<FundInfo>[] | undefined,
-        Array<Option<ITuple<[AccountId, BalanceOf]>>>[] | undefined,
-      ] = await Promise.all([
-        api.query.crowdloan?.funds?.multi<Option<FundInfo>>(paraIds),
-        api.query.slots?.leases?.multi<any>(paraIds),
-      ]);
-
-      const funds = rawFunds ? optFundMulti.transform(paraIds, rawFunds) : [];
-      const leases = optLeaseMulti.transform(paraIds, rawLeases ?? []);
-
-      const minContribution = api?.consts.crowdloan?.minContribution as BlockNumber;
-
-      if (bestNumber && funds && leases) {
-        return createResult(bestNumber, minContribution, funds, leases);
+      if (bestNumber) {
+        return await getFunds(paraIds, bestNumber, api);
```

Comment:
is it important to get new data when `bestNumber` changes? or the data does not change that often?

Reply:
I haven't seen it change at all on polkadot website. I don't think it's really important.